src/en/space-station-14/character-species/guidelines.md

```diff
@@ -1,5 +1,174 @@
+
+### Rule 9: Overpowered Species With Critical Flaws
+
+Species **must not** be balanced so as to be unusually powerful (so as to be strictly, objectively and clearly better in non-trivial ways) if not for a critical flaw reigning in their utility.
+
+These species represent a balancing tightrope for maintainers, especially in a game with constantly-evolving content and balance, and such varied gameplay experiences round-to-round, such as Space Station 14.
+
+An example of being held back by only one thing would be a species that has an unusually high amount of effective health but a major weakness to being stunned. Such extreme designs often work on paper, and may even work in practice on first implementation, but are vulnerable to being exploited by players past the point of being fun. In this example, the weakness to being stunned may be so bad as to make combat against the species entirely one-dimensional, or via the collection and use of specific in-game items and mechanics be evaded entirely, creating a species that requires a draining series of hotfixes and patches to bring back to effective parity.
+
+# Modifying Species After Introduction
+
+From time-to-time, a species that was previously added to Space Station 14 may be proposed for review by the maintenance team. This review would focus on that species' current quality and conformance to the above rules, and propose remediating remedies, if appropriate, to bring the species up to standard.
+
+## Significant Modification to a Species
+
+Additions and improvements to previously-created species are always welcome, but significant modification to a species involves significant modification to players's characters, which may be controversial.
+
+Consultation should be taken from players before deployment of significant re-designs or changes in concept to a round-start species are merged to stable. This may involve seeking feedback from players on the test server about the changes, making players aware of the changes via the Discord, and so on.
+
+## Removal of a Species
+
+A species is the first and most important choice a player makes when building a Space Station 14 character. As such, removing species from the game effectively involves deleting players's characters. This sort of change can be controversial, and must be handled responsibly.
+
+Outright removal of a species that has been deployed outside of an event (such as April Fool's) or a test-merge to Vulture **must be** agreed to by a maintainer vote, after consultation with the Wizard's Den player community. Alternative remediating steps should be considered first, and full deletion should be considered a last resort.
```

Comment:
This is the only time I'm straying onto actual ways-of-working policy changes and it's a deliberate choice. This allows maintainers to safely front "maybe we should remove dwarfs" and allow for a way to handle this without a giant public argument.